entity-registry/src/main/java/com/linkedin/metadata/models/annotation/SearchScoreAnnotation.java

```diff
@@ -0,0 +1,60 @@
+package com.linkedin.metadata.models.annotation;
+
+import com.linkedin.metadata.models.ModelValidationException;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.Nonnull;
+import lombok.Value;
+import org.apache.commons.lang3.EnumUtils;
+
+
+/**
+ * Annotation indicating how the search results should be ranked by the underlying search service
+ */
+@Value
+public class SearchScoreAnnotation {
+
+  public static final String ANNOTATION_NAME = "SearchScore";
+
+  // Name of the field in the search index. Defaults to the field name in the schema
+  String fieldName;
+  // Weight to apply to the score
+  double weight;
+  // Value to use when field is missing
+  double defaultValue;
+  // Modifier to apply to the value. None if empty.
+  Optional<Modifier> modifier;
+
+  public enum Modifier {
```

Comment:
Question - why not expect that the modifier had been applied prior to ingesting the search score?

Reply:
The good thing about doing modifiers on the read path is that you can experiment with using various modifiers without having to reingest the aspects with the specific modifiers. Also counts in general is more objective by nature. Hard to reason about whether or not the feature has been modified on the write path or not.

Reply:
ohhh yes!! makes sense